spec/indico_batch_spec.rb

```diff
@@ -75,7 +76,7 @@
     ])
 
     data = ['Quis custodiet ipsos custodes', 'Clearly english, foo!']
-    response = Indico.batch_language(data, @config)
+    response = Indico.batch_language(data)
```

Comment:
Did we have to remove `config` to get tests to pass?  Or did this just happen unintentionally?